tensorboard/plugins/profile/pod_viewer/topology_graph/topology-graph.ts

```diff
@@ -356,8 +356,8 @@ Polymer({
 
             // Update the tooltip position and value
             d3.select(parent.$.tooltip)
-                .style('left', d3.event.pageX + 10 + 'px')
-                .style('top', d3.event.pageY - 10 + 'px')
+                .style('left', d3.select(this).attr('x') + 'px')
```

Comment:
Wouldn't you want some padding so it shows right (or bottom) of the current item?

Reply:
Done.